tests/unit/src/test/scala/tests/DefinitionSlowSuite.scala

```diff
@@ -84,9 +84,9 @@ object DefinitionSlowSuite extends BaseSlowSuite("definition") {
            |import java.util.concurrent.Future/*Future.java*/ // unused
            |import scala.util.Failure/*Try.scala*/ // unused
            |import org.scalatest.FunSuite/*FunSuite.scala*/
-           |object MainSuite/*L4*/ extends FunSuite/*FunSuite.scala*/ {
+           |object MainSuite/*<no symbol>*/ extends FunSuite/*FunSuite.scala*/ {
```

Comment:
This change means that we were previously returning the symbol itself, whereas now we don't return any symbol. I'm wondering whether we should always fall back to the symbol itself if there are no references.

I'm not feeling strongly about this, just wanted to discuss it, since it's a minor change from the previous behavior.